### DISCUSSION TITLE
Originals Folder is set to a NAS, but indexing uses up all the Diskspace of my Pi

### DISCUSSION THREAD

**Discussion by chrishuette** [photoprism/photoprism#2626] [Q&A]
I was able to install Photoprism in a dockercontainer at my RaspberryPi4 4gb with 128gb SD and to point the originals-Folder via smb to my NAS with 8Tb HDD. There are approx 1.5Tb of Pictures. Indexing these Images makes the Diskspace of my Pi run full to 100%. 

![image](https://user-images.githubusercontent.com/111686554/185789027-68942420-ca51-4e41-b6f1-f70dbf68d821.png)

i tried to find large files and found that the space is taken here:

root/var/lib/docker/volumes/photoprism-originals

![image](https://user-images.githubusercontent.com/111686554/185789390-c1b57bb8-ea93-4c14-8f88-fa426ad3ff00.png)

it looks like photoprism is kopying the originals from the NAS to the Dockercontainer? 

Do i misunderstand the concept? Do i use it wrong? How can i prevent my local disk from running full, or are these just thumbnails and for that amount of 1.5Tb Pictures i simple need a bigger SD Card?

Thanks for Help and appreciate the Work you put into photoprism! IT is a great Tool!

**Comment by graciousgrey** [comment]
In case you let PhotoPrism index your originals directory no copies are created.

https://docs.photoprism.app/user-guide/library/#indexing-originals

PhotoPrism does create thumbnails and sidecar files in the storage directory

https://docs.photoprism.app/user-guide/advanced/folders/#storage

You can configure the location of the storage directory within the docker-compose.yml

You can read more on thumbnails and settings regarding thumbnail creation in our docs:
https://docs.photoprism.app/user-guide/settings/advanced/#what-files-will-be-created-by-photoprism

**Comment by chrishuette** [comment]
I installrd photoprism with a docker-compose file.

the volumes look like this:

![image](https://user-images.githubusercontent.com/111686554/188268494-3cba33dd-6c52-4f44-afca-110ce0dccf74.png)


and this
![image](https://user-images.githubusercontent.com/111686554/188268453-58417016-55aa-4e26-9fea-e9cc9f1bfce8.png)

but the folder /var/lib/docker/volumes/photoprism_photoprism-originals is running full.

i checked the filesizes with the sizes on the NAS and they look as if they are just copied.
![image](https://user-images.githubusercontent.com/111686554/188268664-bd9b2918-cee0-4b8c-885b-6a2804c5cd6b.png)

it is not just thumbnails. The actuall images in the same sizes are taking up space in this originals Folder, wich should just be on the NAS.

What am i missing?

**Comment by reidprichard** [comment]
I believe the "./storage" directory is being used for sidecar images. For example, if you index a bunch of raw files, Photoprism generates JPEGs that can be displayed in your browser. Those generated JPEGs will be put in the "./storage" directory. I don't think there's a reason you can't map "/photoprism/storage" to a NAS rather than using "./storage".
